.dagger/sdk_dotnet.go

```diff
@@ -11,36 +11,49 @@ type DotnetSDK struct {
 	Dagger *DaggerDev
 }
 
-func (t DotnetSDK) Lint(ctx context.Context) error {
+func (t DotnetSDK) Name() string {
+	return "dotnet"
+}
+
+func (t DotnetSDK) CheckLint(ctx context.Context) error {
 	return dag.DotnetSDKDev().Lint(ctx)
 }
 
 func (t DotnetSDK) Test(ctx context.Context) error {
-	installer := t.Dagger.installer("sdk")
-	introspection := t.Dagger.introspection(installer)
 	src := t.Dagger.Source.Directory("sdk/dotnet")
-	return dag.DotnetSDKDev(dagger.DotnetSDKDevOpts{Source: src}).Test(ctx, introspection)
+	return dag.
+		DotnetSDKDev(dagger.DotnetSDKDevOpts{Source: src}).
+		Test(ctx, t.Dagger.introspectionJSON())
 }
 
-func (t DotnetSDK) TestPublish(ctx context.Context, tag string) error {
+func (t DotnetSDK) CheckReleaseDryRun(ctx context.Context) error {
```

Comment:
Can we just remove this function now (and similar non implemented checks), since we have the dynamic check? I think as implemented, it'll show up as a job, but it's actually not doing anything.

Reply:
I think each of the SDKs need to have the same interface for the top level loop to operate